### PR TITLE
release notes 1.9.1

### PR DESCRIPTION
**Pull request by VladimirWork**
Signed-off-by: Vladimir Shishkin <vladimir.shishkin@dsr-corporation.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +46,46 @@
...
+| --- | --- | --- |
+| New DIDs can be created without endorsers | | [INDY-2171](https://jira.hyperledger.org/browse/INDY-2171) |
+| Transaction authors don't need to be endorsers | | [INDY-2173](https://jira.hyperledger.org/browse/INDY-2173) |
+| Grab pool data for failed system tests | | [INDY-2141](https://jira.hyperledger.org/browse/INDY-2141) |
+| Memory profiling needs to be removed from GET_VALIDATOR_INFO output | | [INDY-2182](https://jira.hyperledger.org/browse/INDY-2182) |
+| When view change takes too long instance change should be sent periodically | | [INDY-2143](https://jira.hyperledger.org/browse/INDY-2143) |
+| Implement PBFT viewchanger service with most basic functionality | | [INDY-2147](https://jira.hyperledger.org/browse/INDY-2147) |
+| Extract Orderer service from Replica | | [INDY-2136](https://jira.hyperledger.org/browse/INDY-2136) |
+| Extract and integrate ConsensusDataProvider from Replica | | [INDY-2139](https://jira.hyperledger.org/browse/INDY-2139) |
+| TAA acceptance should use date, not time | | [INDY-2157](https://jira.hyperledger.org/browse/INDY-2157) |
```
Comment (ashcherbakov) [inline CHANGELOG.md]: `TAA acceptance should use date, not time` - this needs to be mentioned in `Major Changes` section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -44,6 +46,46 @@
...
+| Need to make "reask_ledger_status" repeatable | | [INDY-2112](https://jira.hyperledger.org/browse/INDY-2112) |
+
+#### Changes and Additions
+| Description | Additional Information | Ticket Number |
+| --- | --- | --- |
+| New DIDs can be created without endorsers | | [INDY-2171](https://jira.hyperledger.org/browse/INDY-2171) |
+| Transaction authors don't need to be endorsers | | [INDY-2173](https://jira.hyperledger.org/browse/INDY-2173) |
+| Grab pool data for failed system tests | | [INDY-2141](https://jira.hyperledger.org/browse/INDY-2141) |
+| Memory profiling needs to be removed from GET_VALIDATOR_INFO output | | [INDY-2182](https://jira.hyperledger.org/browse/INDY-2182) |
+| When view change takes too long instance change should be sent periodically | | [INDY-2143](https://jira.hyperledger.org/browse/INDY-2143) |
```
Comment (ashcherbakov) [inline CHANGELOG.md]: This can be mentioned in `Major Fixes` section

File: CHANGELOG.md (hunk 1 above)
Comment (VladimirWork) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (VladimirWork) [reply CHANGELOG.md]: Done.

**Pull request merged**